motoman_driver/src/industrial_robot_client/joint_trajectory_streamer.cpp

```diff
@@ -276,8 +343,43 @@ void JointTrajectoryStreamer::streamingThread()
         ROS_WARN("Failed sent joint point, will try again");
 
       break;
+
+    case TransferStates::POINT_STREAMING:
+      ROS_INFO("I'm streaming, sir.");
+      // if no points in queue, streaming complete, set to idle.
+      if (this->streaming_queue_.empty())
+      {
+        ROS_INFO("Point streaming complete, setting state to IDLE");
+        this->state_ = TransferStates::IDLE;
+        break;
+      }
+      // if not connected, reconnect.
+      if (!this->connection_->isConnected())
```

Comment:
Is this reconnection logic required?  Why doesn't [this](https://github.com/ros-industrial/motoman/pull/88/files#diff-f4bf6d1e82f3b9e996488d2548de00fcR324) logic handle it?